events/importer/kulke.py

```diff
@@ -1019,3 +1023,328 @@ def import_keywords(self):
                     data_source=self.data_source,
                     publisher=self.organization,
                 )
+
+    @staticmethod
+    def generate_documentation_md(
+        output_file: Optional[str] = None,
+    ) -> None:  # noqa: C901
```

Comment:
From reusability and testing POV it would be better if there was a function that returned the documentation string instead of printing it.

Reply:
If returning the document as a string, where the printing or output to file should happen?

Reply:
That could happen in the doc generation management command. I agree that it's better for reusability, so I'll change it.